19-10-2020/main.js

```diff
@@ -0,0 +1,133 @@
+// Bài 1
+// Viết một chương trình nhập lương nhân viên, tính thuế thu nhập và lương ròng(số tiền lương thực sự mà nhân viên đó nhận được).Với các thông số giả sử như sau(không theo luật lương, chỉ là con số giả sử để dễ tính toán):
+// 30 % thuế thu nhập nếu lương là 15 triệu.
+// 20 % thuế thu nhập nếu lương từ 7 đến 15 triệu.
+// 10 % thuế thu nhập nếu lương dưới 7 triệu.
+const getSalary = (salary) => {
+    if (salary >= 15000000) {
+        console.log('Thuế thu nhập', salary * 0.3)
+        console.log('Lương ròng   ', salary * 0.7)
+    } else if (salary >= 7000000) {
+        console.log('Thuế thu nhập', salary * 0.2)
+        console.log('Lương ròng   ', salary * 0.8)
+    } else {
+        console.log('Thuế thu nhập', salary * 0.1)
+        console.log('Lương ròng   ', salary * 0.9)
+    }
+}
+
+// Bài 2
+// Viết chương trình nhập tuổi và in ra kết quả nếu tuổi học sinh đó không đủ điều kiện vào học lớp 10. Biết tuổi vào lớp 10 của học sinh là 16.
+const checkAge = (age) => {
+    if (age >= 16) {
+        console.log('Đủ điều kiện vào lớp 10')
+    } else {
+        console.log('Chưa đủ điều kiện vào lớp 10')
+    }
+}
+
+// Bài 3
+// Viết chương trình nhập một số nguyên bất kỳ và in kết quả ra màn hình để thông báo cho người dùng biết số đó lớn hay nhỏ hơn 100.
+const checkNumber = (number) => {
+    if (number > 100) {
+        console.log('Số đã nhập lớn hơn 100')
+    } else if (number < 100) {
+        console.log('Số đã nhập nhỏ hơn 100')
+    } else {
+        console.log('Số đã nhập là 100')
+    }
+}
+
+// Bài 4
+// Viết chương trình nhập vào 3 số nguyên và tìm số lớn nhất trong 3 số đó.
+const maxNumber = (firstNumber, secondNumber, thirdNumber) => {
+    let max = firstNumber
+    if (max < secondNumber) {
+        max = secondNumber
+    }
+    if (max < thirdNumber) {
+        max = thirdNumber
+    }
+    console.log('Số lớn nhất là', max)
+}
+
+// Bài 5
+// Viết chương trình xếp hạng học lực của học sinh dựa trên các điểm bài kiểm tra, điểm thi giữa kỳ, điểm thi cuối kỳ.Nếu:
+// Điểm trung bình >= 9.0 là hạng A.
+// Điểm trung bình >= 7.0 và < 9.0 là hạng B
+// Điểm trung bình >= 5.0 và < 7.0 là hạng C
+// Điểm trung bình < 5.0 là hạng F
+const getRank = (score1, score2, score3) => {
+    let score = (score1 + score2 + sccore3) / 3;
```

Comment:
```suggestion
    const score = (score1 + score2 + sccore3) / 3;
```
`score` có bị thay đổi ko mà đặt là `let`???